src/main/java/org/zalando/nakadi/domain/Subscription.java

```diff
@@ -18,6 +20,8 @@ public Subscription(final String id, final DateTime createdAt, final Subscriptio
 
     private DateTime createdAt;
 
+    private List<SubscriptionEventTypeStats> stats;
```

Comment:
maybe you can add `@JsonInclude(Include.NON_NULL)` here? Because in other case it will show up for all endpoints where we return subscription.